tests/webp-uploads-test.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Tests for webp-uploads module.
+ *
+ * @package performance-lab
+ */
+
+class WebP_Uploads_Tests extends WP_UnitTestCase {
+
+	/**
+	 * Test if webp-uploads is loaded and filter applied when system supports WebP.
+	 */
+	function test_webp_uploads_is_loaded() {
+		// Test before enabling.
```

Comment:
testing that the filter is added is probably sufficient - core already includes extensive tests around the functionality of the `image_editor_output_format` filter.